client.js

```diff
@@ -0,0 +1,29 @@
+const { Client } = require('pg');
+
+const pgclient = new Client({
+  host: process.env.TEST_POSTGRES_HOST,
+  port: process.env.TEST_POSTGRES_PORT,
+  user: 'postgres',
+  password: 'postgres',
+  database: 'postgres',
+});
+
+pgclient.connect();
+
+const table = 'CREATE TABLE users(id SERIAL PRIMARY KEY, username VARCHAR(120), isAdmin boolean)';
+const text = 'INSERT INTO users(username, isAdmin) VALUES($1, $2) RETURNING *';
+const values = ['Eric', false];
+
+pgclient.query(table, (err, res) => {
+  if (err) throw err;
+});
+
+pgclient.query(text, values, (err, res) => {
+  if (err) throw err;
+});
+
+pgclient.query('SELECT * FROM users', (err, res) => {
+  if (err) throw err;
+  console.log(err, res.rows); // Print the data in student table
+  pgclient.end();
+});
```

Comment:
This file was created to automate the database creation in the CI environment. This was here mainly to test if the environment was connected to the container

Reply:
This is likely to cause problems once migrations are implemented. It seems like a lot of rework, what was the thought process here?

Reply:
The original thoughts for this file were to create a script that would allow me to use and test the connection of the Postgres container. When Eric and I were working on this, we were trying to get it up and running.  We got this method from docs. 

https://docs.github.com/en/actions/using-containerized-services/creating-postgresql-service-containers#testing-the-postgresql-service-container

Reply:
I ended up deleting the file and the job; it was there mostly for troubleshooting reasons.